server/controllers/imgUpload.js

```diff
@@ -0,0 +1,23 @@
+const ImageUrl = require("../models/ImageUrl");
+const asyncHandler = require("express-async-handler");
+const fs = require("fs");
+const util = require("util");
+const unlinkFile = util.promisify(fs.unlink);
+const { uploadImages } = require("../s3");
+
+
+// @route POST /images/upload
+// @desc upload images to server
+// @access Public
+exports.uploadImages = asyncHandler(async (req, res) => {
+  const files = req.files;
+  const results = await Promise.all(uploadImages(files));
+  results.forEach(async (item) => {
```

Comment:
Array.prototype.forEach is not designed for asynchronous code

Ref: https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971